src/database/queries/baseFields/selectAll.sql

```diff
@@ -1,2 +1,18 @@
-SELECT base_field_to_json(base_fields.*) as "object"
-FROM base_fields;
+SELECT json_build_object(
```

Comment:
You should edit the `base_field_to_json` function to add the `localizations` rather than modify this query. That way any time a base field is collected it populates the deep value; it also prevents you from having to re-implement the rest of the `base_field_to_json` logic (which would now be in two places if we kept this implementation)

For this you would use the new `base_field_localization_to_json` function that you've already defined.

See the `proposals_to_json` function for an example of a function that selects and builds out a related document into the json result.

This specific file will remain unchanged.

Reply:
Roger this! This makes sense as it ensures that anytime a basefield is returned by the api, it has the localizations field populated. Before there were some endpoints that would not return the localizations, which does seem like poor standardization